File: Fines2Model3/Data_/Sede.cs
using SqlOrganize.Sql.Fines2Model3;
using SqlOrganize.Sql;
using SqlOrganize.CollectionUtils;
using Newtonsoft.Json;
using SqlOrganize.ValueTypesUtils;

namespace SqlOrganize.Sql.Fines2Model3
{
    public partial class Sede
    {
        public override string? Label
        {
            get
            {
                if (!_Label.IsNoE())
                    return _Label;

                return (numero ?? "?") + " " + (nombre ?? "?") + " " + (domicilio_?.Label ?? "?");
            }
            set
            {
                _Label = value;
                NotifyPropertyChanged(nameof(Label));
            }
        }

    }
}

File: Model/Sede_.php
<?php

namespace Fines2;

use \Fines2\Sede;

use SqlOrganize\Sql\DbMy;

class Sede_ extends Sede
{
    

    public ?string $designaciones = null;
    
    public function getLabel(): string {
        return $this->nombre . " (" . $this->numero . ")";
    }




}

